dm/dm.go

```diff
@@ -142,6 +161,10 @@ func execute() {
 
 	switch args[0] {
 	case "templates":
+		if len(args) < 2 {
+			fmt.Fprintln(os.Stderr, "No registry name supplied")
```

Comment:
I don't think this is part of my change? I just check to make sure that there's an argument and give an error message, I think without this check it will just fail with segfault?